validator/checks/url.py

```diff
@@ -107,18 +107,18 @@ def _make_request(self, url):
             if res:
                 res.close()
```

Comment:
Did you see any resource warnings? This might need to be `await res.close()`?

Reply:
Should be replaced by:
```
    async def _make_request(self, url):
        try:
            logging.info('checking {}'.format(url))
            async with aiohttp.request('get', url, headers=self._headers) as response:
                return response.status
        except Exception:
            logging.error('Error making request to %s', url)
            return 500
```
Requires to mock `async with`/`__aexti__` (asynctest's MagicMock can do this).

Laying this off, to prioritize Unlistet project.